### PR TITLE
Add triggers to drivetrain brake/coast mode

### PR DESCRIPTION
**Pull request by katzuv**
- Add function to module IO that sets the brake mode
- Remove setting brake mode in SPARK MAX module IO init
- Implement set idle mode in SPARK MAX module IO
- Remove setting brake mode in Talon FX module IO
- Implement set idle mode in Talon FX module IO
- Add method that sets the idle mode to the swerve module "subsystem" file
- Add private method to swerve that sets the idle mode
- Add command to swerve that sets brake mode
- Add command to swerve that sets coast mode
- Set swerve drive to brake mode when the robot enters autonomous or teleop modes
- Set swerve drive to coast mode after the robot has been disabled for at least seven seconds

Credit: https://bovlb.github.io/frc-tips/coast-mode/

### CODE REVIEW

File: src/main/kotlin/frc/robot/subsystems/swerve/ModuleIO.kt [hunk 1]
```diff
@@ -37,6 +37,8 @@ interface ModuleIO {
 
     fun setVoltage(volts: Double) {}
 
+    fun setIdleMode(isBreakMode: Boolean) {}
```
Comment (rakrakon) [inline src/main/kotlin/frc/robot/subsystems/swerve/ModuleIO.kt:40]: Why not pass in NeutralModeValue instead of a boolean?

Comment (katzuv) [reply src/main/kotlin/frc/robot/subsystems/swerve/ModuleIO.kt:40]: Because the function should work for every motor type.

Comment (rakrakon) [reply src/main/kotlin/frc/robot/subsystems/swerve/ModuleIO.kt:40]: Oh right, forgot we have a SparkMaxIO

**Review by rakrakon — APPROVED** [review]

**Comment by katzuv** [comment]
Thanks Barak!

**Pull request merged**